rules-tests/TypeDeclaration/Rector/ClassMethod/StrictStringParamConcatRector/Fixture/nullable_param_from_null_compare.php.inc

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\StrictStringParamConcatRector\Fixture;
+
+class NullableParamFromNullCompare
+{
+    public static function getOrderBy($orderby, $direction = 'ASC', $default = '')
+    {
+        if (null !== $orderby && ! empty($orderby)) {
```

Comment:
will `is_null()` calls also be covered?

Reply:
Yes, I tried:

```php
if (! is_null($orderby) && ! empty($orderby)) {
     return $orderby.' '.$direction;
}
```

and it also changed to nullable string `?string` 👍